palantir-java-format/build.gradle

```diff
@@ -10,6 +10,9 @@ dependencies {
     api 'com.google.guava:guava'
     implementation 'com.google.errorprone:javac-shaded'
     implementation 'org.functionaljava:functionaljava'
+    implementation 'com.fasterxml.jackson.core:jackson-databind' // purely for the debugger
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8'
+    implementation 'com.fasterxml.jackson.module:jackson-module-parameter-names'
```

Comment:
@dansanduleac I think I'd like to keep the dependencies of palantir-java-format as minimal as possible - thoughts on keeping these in the `:debugger` project and taking a compileOnly dependency on that project?